dotCMS/src/main/java/com/dotcms/rendering/velocity/directive/ContentletDetail.java

```diff
@@ -58,10 +59,18 @@ else if(type.baseType() == BaseContentType.WIDGET  && !Config.getBooleanProperty
       throw new ResourceNotFoundException("cannnot find contentlet id " +  argument + " lang:" + params.language);
     }
 
+    // If the time machine date is after the Publish date of the Contentlet (identifier data)
+    // we need to show the working. If not we only show the live.
+
+    boolean showWorking = false;
+
+    if (context.get("_show_working_") != null && (boolean)context.get("_show_working_")) {
```

Comment:
you can do something like
```
final boolean showWorking = ConversionUtils.toBoolean("_show_working_", context, false);
```
Also "_show_working_" should be a contanst

Reply:
done